frontend/packages/component-library/src/dialog/customDialog.module.scss

```
@@ -8,6 +9,7 @@
   flex-direction: column;
   justify-content: center;
   align-items: center;
+  z-index: variables.$zindex-dropdown;
```

nit: looks like this should be vice versa, 
```suggestion
  z-index: variables.$zindex-modal-backdrop;
```